usaspending_api/common/tests/test_limitable_serializer.py

```diff
@@ -0,0 +1,52 @@
+import pytest
+import json
+from datetime import date
+
+from model_mommy import mommy
+from rest_framework import status
+
+from usaspending_api.awards.models import Award
+from usaspending_api.common.api_request_utils import FilterGenerator
+
+
+@pytest.fixture
```

Comment:
Thanks for rounding out the test cases in `common`--much needed, and great to see!